config/config-observability.yaml

```diff
@@ -99,6 +99,11 @@ data:
     logging.revision-url-template: |
       http://localhost:8001/api/v1/namespaces/knative-monitoring/services/kibana-logging/proxy/app/kibana#/discover?_a=(query:(match:(kubernetes.labels.knative-dev%2FrevisionUID:(query:'${REVISION_UID}',type:phrase))))
 
+    # If non-empty, this enables queue proxy writing request logs to stdout.
+    # The value determines the shape of the request logs and it must be a valid go text/template.
+    # Entering an invalid value here will turn off the feature.
+    logging.request-log-template: '{"httpRequest": {"requestMethod": "{{.Request.Method}}", "requestUrl": "{{js .Request.RequestURI}}", "requestSize": "{{.Request.ContentLength}}", "status": {{.ResponseCode}}, "responseSize": "{{.ResponseSize}}", "userAgent": "{{js .Request.UserAgent}}", "remoteIp": "{{js .Request.RemoteAddr}}", "serverIp": "{{env "SERVING_POD_IP"}}", "referer": "{{js .Request.Referer}}", "latency": "{{.ResponseLatency}}s", "protocol": "{{.Request.Proto}}"}, "traceId": "{{index .Request.Header "X-B3-Traceid"}}"}'
```

Comment:
I would use the following syntax for this to make it more readable (after all this is user documentation!):

```
logging.request-log-template: |
  {  
     "httpRequest":{  
        "requestMethod": "{{.Request.Method}}",
        "requestUrl": "{{js .Request.RequestURI}}",
        "requestSize": "{{.Request.ContentLength}}",
        "status": "{{.ResponseCode}}",
        "responseSize": "{{.ResponseSize}}",
        "userAgent": "{{js .Request.UserAgent}}",
        "remoteIp": "{{js .Request.RemoteAddr}}",
        "serverIp": "{{env "SERVING_POD_IP"}}",
        "referer": "{{js .Request.Referer}}",
        "latency": "{{.ResponseLatency}}s",
        "protocol": "{{.Request.Proto}}"
    },
    "traceId":"{{index .Request.Header "X-B3-Traceid"}}"
  }
```

I think you made a comment about things showing up as a single line, perhaps that is part of what's going on here. I'm just wondering how we make this a big more approachable.

Reply:
It needs to be single line. I can probably use > instead of |. Will try that.

Reply:
>- or > didnt work. They all expand to multiple lines. Will keep this as is.